src/main/java/com/faforever/client/fa/LaunchCommandBuilder.java

```diff
@@ -169,6 +182,17 @@ public LaunchCommandBuilder rehost(boolean rehost) {
       command.add(localIp + ":" + localGpgPort);
     }
 
+
+    if(avatarUrl != null) {
+	  String urlTOfilename = null; // In lobby it is easier to use the name of the avatar file than the link itself.
+	  urlTOfilename = Paths.get(URI.create(avatarUrl).getPath()).getFileName().toString();
```

Comment:
why do you first create and then assign inside the same block?
(multiple times)

Reply:
@Geosearchef Like I tried to do so, but the Travis (plugin in github) gave an error.

Reply:
Pls do it again and let me take a look. I can't believe that, maybe there was something else going wrong. If it really happened due to this we need to fix travis, not the code ;).